### DISCUSSION TITLE
WalletConnect V2 QRcode Browser Javascript not shown

### DISCUSSION THREAD

**Discussion by arculusredpath** [WalletConnect/walletconnect-monorepo#2752] [Developer Support] [answered]
The QRcode in the latest connect for Wallet Connect V2 does not display and hangs really using latest javascript
This is a very simple example.

**Code is provided and works with  2.4.4**
     unpkg.com/@walletconnect/ethereum-provider@2.4.4

**and does not work anymore with 2.8.1 or 2.8.2**
https://unpkg.com/@walletconnect/ethereum-provider@2.8.2/dist/index.umd.js

**The QRcode is not shown to connect.**
Simply edit the bug.txt to set the project ID and save as bug.html and load into a Browser.
Change the script from 2.4.4 to 2.8.2 and does not work.

[bug.txt](https://github.com/WalletConnect/walletconnect-monorepo/files/11857092/bug.txt)
![bug](https://github.com/WalletConnect/walletconnect-monorepo/assets/111876181/fee8085d-cba9-462e-8c5d-95f4b20e4c26)

**Comment by nicktan147** [comment]
Did you managed to resolve this issue?

**Reply by arculusredpath** [reply, marked as answer]
Yes add
   showQrModal: true,
to the
  await provider.initialize({
and works was it is did win previous versions.
